app/models/preneeds/burial_form.rb

```diff
@@ -58,8 +99,14 @@ def as_eoas
       Common::HashHelpers.deep_compact(hash)
     end
 
-    def self.validate(schema, form, root = 'application')
-      JSON::Validator.fully_validate(schema, { root => form&.as_json }, validate_schema: true)
+    # @return [Array<String>] array of strings detailing schema validation failures. empty array if form is valid
+    #
+    def validate
+      JSON::Validator.fully_validate(
+        VetsJsonSchema::SCHEMAS[FORM],
+        { 'application' => self&.as_json },
+        validate_schema: true
+      )
```

Comment:
This is the refactor mentioned in the comment in the controller.  Can you think of any reason not to do this?